src/youtube/js/youtube.js

```diff
@@ -366,9 +366,13 @@ require("../../../html5-common/js/utils/constants.js");
         youtubeID = '';
       }
       player = null;
-      youtubePlayer.destroy();
-      youtubePlayer = null;
+      if (youtubePlayer) {
+        youtubePlayer.destroy();
+        youtubePlayer = null;
+      }
       playerReady = false;
+      clearInterval(timeUpdateInterval);
```

Comment:
But I added checks for timeUpdateInterval before it is being cleared in a few places in this file